.dictionary.txt

```diff
@@ -1,5 +1,3 @@
-!preform
-containername
 filesonly
 Laven
 setuptools
```

Comment:
I may be missing something, but at [face value](https://github.com/streetsidesoftware/cspell-dicts/blob/91afe6a2ef9c55b8b5fd7eeaea130a76f5c5098c/dictionaries/python/src/common/extra.txt#L604), this word looks to be already present upstream.

Reply:
Evidently I was missing something, although I'm not sure what.